streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -124,9 +112,9 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", RegionCount.class);
         regionCountSerializer.configure(serdeProps, false);
 
-        KStream<String, PageView> views = builder.stream(stringDeserializer, pageViewDeserializer, "streams-pageview-input");
+        KStream<String, PageView> views = builder.stream(Serdes.String(), Serdes.serdeFrom(null, pageViewDeserializer), "streams-pageview-input");
```

Comment:
See also [my other comment](https://github.com/apache/kafka/pull/1066/files#r56475404) above (and the gist idea at https://gist.github.com/miguno/b1fce35db880c6afa7b6).